### PR TITLE
adding invoices query

### PR DESCRIPTION
**Pull request by sbeltaine**
adding invoices query

### CODE REVIEW

**Comment by sbeltaine** [comment]
First draft of the invoices query in development.

**Comment by adoerrer** [comment]
This is a first draft and still in development

**Comment by nassibnassar** [comment]
Note that this query requires invoice data to be present, e.g. in `folio_snapshot_20201105`.

File: sql/report_queries/costs/invoices.sql [hunk 1]
```diff
@@ -0,0 +1,151 @@
+/**
+
+PURPOSE
+
+This query provides a summary of what was spent by purchase order line  
+on an invoice line across all the invoices within a particular time period. 
+Invoice line adjustments are included in the invoice line total. 
+This query only shows invoice line data that is linked to a purchase order.
```
Comment (nassibnassar) [inline sql/report_queries/costs/invoices.sql:8]: Could this be moved to a `README.md` file?

**Review by nassibnassar — CHANGES_REQUESTED** [review]

File: sql/report_queries/costs/invoices.sql (hunk 1 above)
Comment (adoerrer) [reply sql/report_queries/costs/invoices.sql:8]: Yes, I think so. It was thought as a working draft too and not loose any thoughts on it.

Comment (sbeltaine) [reply sql/report_queries/costs/invoices.sql:8]: I will create readme for the invoices query and move the documentation from the query into that file.

**Review by adoerrer — APPROVED** [review]
Good to go for a first version!

**Review by nassibnassar — APPROVED** [review]

**Comment by nassibnassar** [comment]
Great!

**Pull request merged**